### PR TITLE
Find and Replace

### PR DESCRIPTION
**Pull request by ArifPMan**
Hi, Prashant,

Please review and provide your feedback for this feature. If you're okay with the content, please approve the PR.

### CODE REVIEW

File: v6/postman/find_replace.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+title: "Find and Replace"
+page_id: "findnreplace"
+warning: false
+---
+
+### Find and Replace in Postman
+
+Finding and replacing text in your API projects is quick and easy in Postman. The Postman app enables you to perform a global find and replace operation that works seamlessly across its various components like Collections, Environments, Globals, and Open tabs. Postman's find and replace functionality supports both simple text-based search and replace as well as advanced search and replace that lets you choose entities like Collections, Environments, Globals, and Tabs. 
+
+You can easily launch the Find and replace panel. To do that, in the Postman's main interface, click the find icon at the bottom of the sidebar, as illustrated in the screen below:
```
Comment (prashantagarwal) [inline v6/postman/find_replace.md]: @ArifPMan We also have a shortcut key to open find and replace. Cmd + Shift + F (for mac) and
Ctrl + Shift + F (for windows and linux).

File: v6/postman/find_replace.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Find and Replace"
+page_id: "findnreplace"
+warning: false
+---
+
+### Find and Replace in Postman
+
+Finding and replacing text in your API projects is quick and easy in Postman. The Postman app enables you to perform a global find and replace operation that works seamlessly across its various components like Collections, Environments, Globals, and Open tabs. Postman's find and replace functionality supports both simple text-based search and replace as well as advanced search and replace that lets you choose entities like Collections, Environments, Globals, and Tabs.
```
Comment (prashantagarwal) [inline v6/postman/find_replace.md]: We can rephrase the last line to remove simple text-based search and replace. We can add more emphasis on the customisability of search

File: v6/postman/find_replace.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Click the find icon to launch the find and replace panel, as illustrated in the screen below:
+
+[![find replace panel](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Find_replace3.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Find_replace3.png)
+
+The Find and Replace panel also contains options for toggling case sensitivity, performing regular expression matching and scoping the search to selections.
+
+Let's perform a search operation and find and replace a string 'pstman' by keeping the scope to 'everything'. This is illustrated in the screen below:
+
+[![find replace results](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Find_replace1.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Find_replace1.png)
```
Comment (prashantagarwal) [inline v6/postman/find_replace.md]: Can be a gif performing showing this operation end to end

**Comment by ArifPMan** [comment]
Hi, Numaan,

Find and replace is done. Prashant has approved this req. You can review and merge the PR.

**Comment by ArifPMan** [comment]
Hi, Numaan,

Find and replace is done. Prashant has approved this req. You can review and merge the PR.

**Comment by ArifPMan** [comment]
Hi, Numaan,

Find and replace is done. Prashant has approved this req. You can review and merge the PR.

File: _data/v6_map.json [hunk 4]
```diff
@@ -453,6 +453,18 @@
...
       {
+        "title": "Find and Replace",
+        "url_prefix": "find_replace",
+        "href":"/docs/v6/postman/find_replace",
+        "external": false
+      },
+      {
+        "title": "Customizing Postman",
+        "url_prefix": "customizing postman",
+        "href":"/docs/v6/postman/customizing postman",
```
Comment (numaanashraf) [inline _data/v6_map.json]:
```suggestion
        "href":"/docs/v6/postman/customizing_postman",
```

File: v6/postman/find_replace.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Find and Replace"
+page_id: "findnreplace"
+warning: false
+---
+
+### Find and Replace in Postman
+
+Finding and replacing text in your API projects is quick and easy in Postman. The Postman app enables you to perform a global find and replace operation that works seamlessly across its various components like Collections, Environments, Globals, and Open tabs. Postman's find and replace functionality supports advanced search and replace functionality that lets you choose entities like Collections, Environments, Globals, and Tabs. This essentially means you have the power to scope your search only to your Collections, Environments, Globals and Tabs.
```
Comment (numaanashraf) [inline v6/postman/find_replace.md]: > The Postman app enables you to perform a global find and replace operation that works seamlessly across its various components like Collections, Environments, Globals, and Open tabs. Postman's find and replace functionality supports advanced search and replace functionality that lets you choose entities like Collections, Environments, Globals, and Tabs. This essentially means you have the power to scope your search only to your Collections, Environments, Globals and Tabs. 

All three sentences are talking about the same point again and again.

Comment (ArifPMan) [reply v6/postman/find_replace.md]: I have updated the content as per your comments Numaan.

File: _data/v6_map.json [hunk 6]
```diff
@@ -453,6 +453,18 @@
...
       },
       {
+        "title": "Find and Replace",
+        "url_prefix": "find_replace",
+        "href":"/docs/v6/postman/find_replace",
+        "external": false
+      },
+      {
+        "title": "Customizing Postman",
+        "url_prefix": "customizing postman",
```
[the remark this answers is not in the document]
Comment (numaanashraf) [reply _data/v6_map.json]: why is customizing postman going in as part of this pull request?

**Comment by ArifPMan** [comment]
Addressed the comments. Please review and approve.

File: v6/postman/find_replace.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+The Find and Replace panel also contains options for toggling case sensitivity, performing regular expression matching and scoping the search to selections.
+
+Let's perform a search operation and find and replace the string 'postman' with 'pstman' by keeping the scope to 'everything'. This is illustrated in the screen below:
+
+[![find replace results](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Find_replace2.gif)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/Find_replace2.gif)
+
+In the above screen, you can see the results that show how the string 'postman' is searched and replaced with 'pstman'. You can limit a search to a subset of the collections/environments/globals/tabs in your API project by selecting the radio button 'Choose entities to find in'. 
+
+**Note:** You can also use shortcut keys to use this feature. Use Cmd + Shift + F for macOS and Ctrl + Shift + F for Windwos and Linux OS.
```
Comment (numaanashraf) [inline v6/postman/find_replace.md]: typo `Windwos`

**Pull request merged**